state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -626,14 +639,31 @@ pub mod pallet {
 		DepositBoosted {
 			deposit_address: TargetChainAccount<T, I>,
 			asset: TargetChainAsset<T, I>,
-			amount: TargetChainAmount<T, I>,
+			amounts: BTreeMap<BoostPoolTier, TargetChainAmount<T, I>>,
 			deposit_details: <T::TargetChain as Chain>::DepositDetails,
+			channel_id: ChannelId,
 			// Ingress fee in the deposit asset. i.e. *NOT* the gas asset, if the deposit asset is
 			// a non-gas asset.
 			ingress_fee: TargetChainAmount<T, I>,
 			boost_fee: TargetChainAmount<T, I>,
 			action: DepositAction<T::AccountId>,
 		},
+		BoostFundsAdded {
+			account_id: T::AccountId,
+			boost_pool: BoostPoolId<T::TargetChain>,
+			amount: TargetChainAmount<T, I>,
+		},
+		AccountStoppedBoosting {
+			account_id: T::AccountId,
+			boost_pool: BoostPoolId<T::TargetChain>,
+			account_available_amount: TargetChainAmount<T, I>,
+			account_in_use_amounts: BTreeMap<BoostId, TargetChainAmount<T, I>>,
+		},
+		InsufficientBoostLiquidity {
+			boost_id: BoostId,
```

Comment:
Is fair enough to include this boost id, but in the spec we have channel_id here too, so need to add that

Reply:
Will there even be a `boost_id` if a boost failed?

Reply:
Yeah, we will need the `channel_id` here so we can link to the deposit.

Reply:
> Will there even be a `boost_id` if a boost failed?

yes - but your question suggests that maybe this naming is confusing. It's really the `prewitness_deposit_id` so perhaps we should just name it that @j4m1ef0rd

Reply:
Yeah i agree. the prewitness_deposit_id becomes a boost id once it is boosted, so we should draw the line there.